content/visit-belgium-tips.adoc

```diff
@@ -0,0 +1,78 @@
+= Tips When Visiting Belgium
+:jbake-type: normalBase
+:jbake-description: Travelling to Belgium for a JUG or conference? These are some of our tips...
+:jbake-priority: 1.0
+:showtitle:
+
+During a call with BeJUG members, we collected a list of exciting places to share with the Java community. We know a JUG or conference schedule allows you to fill your days with a lot of Java, but for those of you coming early, staying longer or just looking for a good local restaurant we got you covered.
```

Comment:
I would add a big note that "we did not get any kind of sponsorship nor deal from any of these places." 

Or something similar. To make it clear we are just posting tips, nothing sponsored.

Reply:
Good remark, note is added